src/SFA.DAS.ApplyService.InternalApi/Startup.cs

```diff
@@ -173,6 +173,12 @@ private void ConfigHttpClients(IServiceCollection services)
             })
             .SetHandlerLifetime(handlerLifeTime);
 
+            services.AddHttpClient<CharityCommissionOuterApiClient, CharityCommissionOuterApiClient>(config =>
```

Comment:
Register the outerapiclient here instead, with headers, like it is done in apply